package.json

```diff
@@ -49,6 +49,7 @@
   },
   "dependencies": {
     "archiver": "^3.0.3",
```

Comment:
Shouldn't we remove it now if we use `bestzip`?

Reply:
I fixed the builds.

Regarding archiver - bestzip actually references archiver itself, so the question is whether you want to specify the version of archiver or allow bestzip to do it for you.

I wanted to minimize my changes when making the PR.

Reply:
I think we can let bestzip to decide which version best suit it